src/app/components/Account/AccountLink.tsx

```diff
@@ -6,16 +6,26 @@ import Link from '@mui/material/Link'
 import { TrimLinkLabel } from '../TrimLinkLabel'
 import { RouteUtils } from '../../utils/route-utils'
 import { Layer } from '../../../oasis-indexer/api'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../styles/theme/colors'
 
-export const AccountLink: FC<{ address: string; layer: Layer }> = ({ address, layer }) => {
+export const AccountLink: FC<{ address: string; layer: Layer; alwaysTrim?: boolean }> = ({
+  address,
+  layer,
+  alwaysTrim,
+}) => {
   const theme = useTheme()
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
   const to = RouteUtils.getAccountRoute(address, layer)
-  return isMobile ? (
-    <TrimLinkLabel label={address} to={to} />
-  ) : (
-    <Link component={RouterLink} to={to}>
-      {address}
-    </Link>
+  return (
+    <Typography variant="mono" component="span" sx={{ color: COLORS.brandDark, fontWeight: 700 }}>
```

Comment:
Do we no longer need `sx={{ color: COLORS.brandDark, fontWeight: 700 }}`? BlockLink doesn't have it

Reply:
I am not sure, I just copied the code as it was used in the current code-base.

I agree that it would probably make sense to make them more similar to each other.